R/ggplotly.R

```diff
@@ -917,11 +917,15 @@ gg2list <- function(p, width = NULL, height = NULL,
     # facet strips -> plotly annotations
     if (has_facet(plot)) {
       col_vars <- ifelse(inherits(plot$facet, "FacetWrap"), "facets", "cols")
-      col_txt <- paste(
-        plot$facet$params$labeller(
-          lay[names(plot$facet$params[[col_vars]])]
-        ), collapse = br()
-      )
+      col_txt <- if (!length(names(plot$facet$params[[col_vars]])) == 0) {
+        paste(
+          plot$facet$params$labeller(
+            lay[names(plot$facet$params[[col_vars]])]
+          ), collapse = br()
+        )
+      } else {
+        ""
+      }
```

Comment:
This style leads to a smaller diff and also matches the style of the subsequent code

```suggestion
      col_txt <- paste(
        plot$facet$params$labeller(
          lay[names(plot$facet$params[[col_vars]])]
        ), collapse = br()
      )
      if (length(names(plot$facet$params[[col_vars]])) == 0) col_txt <- ""
```